docs/config/tech-docs.yml

```diff
@@ -0,0 +1,38 @@
+# Host to use for canonical URL generation (without trailing slash)
+host: docs.teacher-training-courses.service.gov.uk
```

Comment:
Should we use `api2.publish-teacher-training-courses.service.gov.uk` until we have a new domain setup?

Reply:
done